compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -754,6 +754,36 @@ TR::Register *OMR::X86::TreeEvaluator::sloadEvaluator(TR::Node *node, TR::CodeGe
    return reg;
    }
 
+TR::Register *iReadBarrierEvaluator(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
Do you mean `OMR::X86::TreeEvaluator::iReadBarrierEvaluator` ?

Reply:
thanks for catch this. changed accordingly.